main.cpp

```diff
@@ -0,0 +1,127 @@
+#include <iostream>
+#include <fstream>
+
+using std::string;
+using std::cout;
+using std::cin;
+using std::ofstream;
+using std::ifstream;
+
+const string FILE_PATH = "C:\\Users\\Samantha\\ClionProjects\\Assignment2-Config-Files\\Config.txt";
+
+void init() {
+    const int SIZE = 5;
+    const string HEADERS[SIZE] = {"[Name]","[E-mail]","[Password]", "[Timezone Offset]", "[File Path]"};
```

Comment:
I really want to make the read/output in the edit function to work but I'm running into some problems with that. Going to get back to working on it today.